lib/grape/validations/params_scope.rb

```diff
@@ -358,8 +358,12 @@ def validates(attrs, validations)
         # type casted values
         coerce_type validations, attrs, doc, opts
 
+        # There are a number of documentation options on entities that don't have
+        # corresponding validators. Since there is nowhere that enumerates them all,
+        # we maintain a list of them here and skip looking up validators for them.
+        reserved_keywords = %i[as required param_type is_array format example]
```

Comment:
Let's make this a constant `RESERVED_KEYWORDS`. Maybe we should also rename it? These are documentation options so `RESERVED_DOCUMENTATION_KEYWORDS`? WDYT?

Reply:
Sounds good to me!